pkg/helpers/workspace.go

```diff
@@ -11,11 +11,11 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-func ManagedClusterSetNameForWorkspace(workspaceName string) string {
+func ComputeWorkspaceName(workspaceName string) string {
 	// TODO: THIS IS NOT SUFFICIENT AT ALL. Probably doesn't handle all illegal characters and does NOT uniquely identify a workspace.
```

Comment:
Can we update this TODO to reflect the situation now that you've made these changes? Do we handle all illegal characters? Do we uniquely identify a workspace, short of using the persistent identifier once available? Let's be clear where we're landing with these changes in. Thanks!